dags/openshift_nightlies/scripts/install/ocm_gcp.sh

```diff
@@ -55,10 +55,10 @@ setup(){
     export OCM_CLI_VERSION=$(cat ${json_file} | jq -r .ocm_cli_version)
     if [[ ${OCM_CLI_VERSION} == "master" ]]; then
         git clone https://github.com/openshift-online/ocm-cli
-	pushd ocm-cli
-	make
-	sudo mv ocm /usr/local/bin/
-	popd
+        pushd ocm-cli
+        sudo PATH=$PATH:/usr/bin:/usr/local/go/bin make
```

Comment:
Do we need this again after the initial export of $PATH? I don’t think I needed this for rosa

Reply:
`sudo` command doesn't use the local env `PATH`, so I have to set it explicitly for `make`.